### DISCUSSION TITLE
Cannot sync to chain or graph

### DISCUSSION THREAD

**Discussion by rtarnaud** [lightningnetwork/lnd#8918] [Troubleshooting] [answered]
Hi,

I have a LN node running as part of a BTCPayServer installation.
The BTC node is fully synced and I thought the LN one was too since nbxplorer seems to be reporting fine.
However, the LN node is not synced to the chain or the graph:

```json
{
    "version":  "0.18.1-beta commit=basedon-v0.18.1-beta-fresh-btcpay",
    "commit_hash":  "9f94d045ae2123f4f1e9bd9c041bf48332b10502",
    "identity_pubkey":  "027db07175e71e8d1a52c9d6d1813d48111f2292131c73f89c596b45b00bad2d6e",
    "alias":  "directaffinity-staging",
    "color":  "#3399ff",
    "num_pending_channels":  0,
    "num_active_channels":  0,
    "num_inactive_channels":  15,
    "num_peers":  0,
    "block_height":  2868658,
    "block_hash":  "000000000000000e89ff77955aef0a4b5b28bbed9c707cee98d87142f7369311",
    "best_header_timestamp":  "1721207866",
    "synced_to_chain":  false,
    "synced_to_graph":  false,
    "testnet":  true,
    "chains":  [
        {
            "chain":  "bitcoin",
            "network":  "testnet"
        }
    ],
    ...
}
```

I also have a number of inactive channels that I cannot force close but I imagine that I need to sync before that. The node had been unavailable for a few weeks due to a server issue (disk was full and mempool prevented chain to sync) but is back online now.

Does anyone have a clue on what could be the problem?

Thanks!

**Comment by Liongrass** [comment, marked as answer]
How long as the node been in this state? I recently revived an old node on 0.18.1 and it took a full day to regain sync. You can also try restarting, then let it run for a bit and share the logs. This might give some better insights into what is keeping it from syncing
